grml-debootstrap

```diff
@@ -1385,6 +1391,26 @@ mount_target() {
 }
 # }}}
 
+# mount the new partition or if it's a directory do nothing at all {{{
+mount_boot_target() {
+  if [ -n "$MOUNT_BOOT" ] ; then
+     einfo "/boot is not on a separate partition, nothing to mount."
```

Comment:
You can just "return 0" afterwards and don't need the `else ... ` afterwards, since nothing from this function needs to be executed if separate /boot isn't requested.